spec.bs

```diff
@@ -815,6 +820,51 @@ To <dfn>obtain a report's shared info</dfn> given an [=aggregatable report=]
 1. Return the result of [=serializing an infra value to a json string=] given
     |sharedInfo|.
 
+User-agent automation {#user-agent-automation}
+==============================================
+
+A user agent holds a boolean <dfn>automation local testing mode enabled</dfn>
+(default false).
+
+For the purposes of user-agent automation and website testing, this document
+defines the below [[WebDriver]] [=extension commands=] to control the API
+configuration.
+
+Set local testing mode {#set-local-testing-mode}
+------------------------------------------------
+
+<figure id="table-webdriver-setlocaltestingmode" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>HTTP Method</th>
+                <th>URI Template</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>POST</td>
+                <td>`/session/{session id}/private-aggregation/`<dfn
+                    noexport>`localtestingmode`</dfn></td>
+            </tr>
+        </tbody>
+    </table>
+</figure>
+
+The [=remote end steps=] are:
+
+1. If |parameters| is not a JSON [[ECMASCRIPT#sec-json-object|Object]], return a
```

Comment:
The section of the ECMAScript spec this points to is about _the_ JSON object, i.e. `globalThis.JSON`, but that’s not what we’re interested in here. To fix this, link to https://tc39.es/ecma262/#sec-objects instead of `#sec-json-object`. (We could also change the phrasing to “is not an Object” or “is not a JSON-formatted object” to be more precise but that’s not a blocker.)

Relevant precedent in the WebDriver classic spec is e.g. here: https://w3c.github.io/webdriver/#add-cookie

Reply:
Ah good point! Fixed the link and updated the phrasing